Source/RapyutaSimulationPlugins/Private/Core/RRNetworkPlayerController.cpp

```diff
@@ -328,3 +199,49 @@ void ARRNetworkPlayerController::ReceivedPlayer()
         ClientRequestLocalClockUpdate(LocalTime);
     }
 }
+
+void ARRNetworkPlayerController::ServerSetLinearVel_Implementation(ARRBaseRobot* InServerRobot,
```

Comment:
Maybe We can create a replicated ActorComponent that does these Set Velocity actions and attach it to the player controller for the robots to be able to pass these RPCs to? Kind of like how SimulationStateClient is. We could even have one per robot, in case differentiation between the robots in the future is needed but not too sure if that would be a good idea for now

Reply:
sounds good. 
but it may require a bit more time. let me work on different pr.

Reply:
https://www.wrike.com/workspace.htm?acc=1794223#folder/858739231/list?filters=status%3Dactive&sidePanelItemId=955024545&sortOrder=5&spaceId=364052400&viewId=74889003